testsuite/tests/ui/search/test_account_search.py

```diff
@@ -58,20 +73,18 @@ def test_search_multiple_accounts(login, navigator, custom_account, request):
         - you search account by specific name it will return the correct one
     """
     name = blame(request, "")
-    params = [dict(org_name=f"{name}_name", username="username", email="email@anything.invalid", password="123456"),
-              dict(org_name=f"{name}_name2", username="username2", email="email2@anything.invalid", password="123456"),
-              dict(org_name=f"{name}", username="user", email="mail@anything.valid", password="123456"),
-              dict(org_name=blame(request, "RedHat"), username="random", email="random@random.random",
-                   password="123456")
-              ]
-    for param in params:
-        custom_account(param)
+    custom_account(name)
+    custom_account(f"{name}_name1")
+    custom_account(f"{name}_name2")
+    custom_account("RedHat")
```

Comment:
Imho some name which does not contain RedHat should be better because when you run the test with RHSSO tests(or manual using a RedHat email address) they by default create the organization RedHat. So if tests for RHSSO fail, resources won't be cleaned and there may be unexpected results.